app.js

```diff
@@ -34,7 +36,20 @@ app.use(bodyParser.urlencoded({extended: false}));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
-app.use('/messages', routes);
+app.use('/messages', messagesRoute);
+
+app.use('/long/messages', longPollingMessagesRoute);
+
+//timeout catcher
+app.use(function (req, res, next) {
```

Comment:
The order is given by the order the order of the app.use methods. that means a request will first try and match /message and /long/messages route and afterwards (if a response is not already sent) it'll be forwarded to the timeout catcher and the error handler